gap/elements/bipart.gi

```diff
@@ -93,6 +93,14 @@ InstallGlobalFunction(Bipartition,
 function(classes)
   local n, copy, i, j;
 
+  n := Sum(classes, Length) / 2;
+
+  if n >= 2 ^ 31 then
+    ErrorNoReturn("Semigroups: Bipartition: usage,\n",
+                  "the argument <classes> must be a list of lists whose union",
+                  "has length at most 2 ^ 31 - 1,");
```

Comment:
shouldn't this be "consists of values in the range 1 to 2 ^ 31 - 1"? Since if it is a list of length 10 containing numbers in excess of 2 ^ 31 - 1, then it will not work.